cishouseholds/derive.py

```diff
@@ -16,6 +19,37 @@
 from cishouseholds.pyspark_utils import get_or_create_spark_session
 
 
+def assign_column_value_from_multiple_column_map(
+    df: DataFrame, column_name_to_assign: str, column_name_to_map: List[List[Any]], column_names: List[str]
+):
+    """
+    assign column value based on values of any number of columns in a dictionary
+    Parameters
+    ----------
+    column_name_to_assign
+    column_name_to_map
```

Comment:
The name of the `column_name_to_map` parameter doesn't seem correct given the description says these are value options. These are lists rather than dictionaries, so please elaborate on how the output value is determined

Reply:
You could add an example to the docstring if that helps to explain, but I think it's important to have this well documented given how complex the inputs look below